### PR TITLE
update client_id and identity number

### LINKED ISSUE #200: Client ID wrong
Citizen officers don't care about or understand our primary keys in our database. Displaying that is potentially confusing. The user will recognise only immigation NZ's client numbers

Change from using our internal database primary keys `client.id`, to instead use immigration's client id `client.im_client_id`

### PR DESCRIPTION
**Pull request by mischa-s**
closes #200

### CODE REVIEW

File: app/javascript/bundles/Identity/components/KnownIdentity.jsx [hunk 1]
```diff
@@ -26,7 +27,7 @@ export default class KnownIdentity extends React.Component {
         <div className="left">
           <Table>
             <TableRow headerLabel="ID No.">
-              <b>{id}</b>
+              <b>{identity_number}</b>
```
Comment (Br3nda) [inline app/javascript/bundles/Identity/components/KnownIdentity.jsx]: what's in {identity_number}?

Comment (mischa-s) [reply app/javascript/bundles/Identity/components/KnownIdentity.jsx]: it is the second item in the identities array list in that picture of the data you showed, is that not what you meant?

Comment (Br3nda) [reply app/javascript/bundles/Identity/components/KnownIdentity.jsx]: We're removing identity number
Replacing with passport number


image

Comment (dlouise64) [reply app/javascript/bundles/Identity/components/KnownIdentity.jsx]: there is identitiy_number and serial_number

Comment (Br3nda) [reply app/javascript/bundles/Identity/components/KnownIdentity.jsx]: have you kept the old variable name.. ? I'm way confused why identity_number is in there still.

File: app/javascript/bundles/Identity/components/KnownIdentity.jsx [hunk 2]
```diff
@@ -26,7 +27,7 @@ export default class KnownIdentity extends React.Component {
         <div className="left">
           <Table>
             <TableRow headerLabel="ID No.">
```
Comment (Br3nda) [inline app/javascript/bundles/Identity/components/KnownIdentity.jsx]: I expected this to change to "Passport Number"

**Pull request merged**